java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -93,8 +93,12 @@ private void parallelAnnotation(JSONObject op, StringBuilder sb) {
         if (parallel != null && parallel) {
             sb.append("@parallel(width=");
             Object width = op.get("width");
-            if (width instanceof Integer)
+            if (width instanceof Integer) {
                 sb.append(Integer.toString((int) width));
+        	}
```

Comment:
It seems that the python width parameter has the type of long when written to the JSON.  I could not figure out how to force it to integer.
Using source line:
      topo = Topology("test_TopologyParallel")
      hw = topo.source(test_functions.hello_world)   
      hwp = hw.parallel(4)
      :